nilearn/plotting/html_surface.py

```diff
@@ -130,6 +208,8 @@ def _fill_html_template(info, embed_js=True):
 def view_img_on_surf(stat_map_img, surf_mesh='fsaverage5',
                      threshold=None, cmap=cm.cold_hot,
                      black_bg=False, vmax=None, vmin=None, symmetric_cmap=True,
+                     bg_map='auto', bg_on_data=False,
+                     bg_map_rescale="auto", darkness=.5,
```

Comment:
`bg_map_rescale` has no effect. Indeed, the only possible bg map is the curvature sign, so values are in `{-1, 1}`. rescaling replaces `-1` with `0`, but the matplotlib colormap truncates the values outside of [0, 1] so it already does this anyway

Reply:
`bg_map="auto"` means we'll use the curvature sign by default, but users can actually give the fsaverage background map they want here, so I think it still makes sense to have `bg_map_rescale`. What do you think?

Reply:
Maybe what you are implicitly saying is that `bg_map` should either be `"auto`" or `None`, as one of your other comments suggests, which limits the number of possibilities for `bg_map_rescale`?

Reply:
In my humble opinion, it's an interesting feature to let users choose the background map they want to use. Maybe it's not very explicit here how to set it, but at least it's something they can do.

Reply:
I definitely agree it would be a useful feature but in the case of `view_img_on_surf` how would that work?

if it is not `"auto"`, `bg_map` is used as-is, for _both hemispheres_ [here](https://github.com/nilearn/nilearn/pull/3173/files#diff-db0386bc2cff240a12444bc9eaf0c6f45ee41abc63687a42e4e700586c3a9e58R179) -- so if we pass a numpy array or surface data file we would use the same bg map for both hemispheres which doesn't make sense:

![rh](https://user-images.githubusercontent.com/9196501/210827890-1407374c-9a2e-47b3-a1a2-88fca2d03b4c.png)

what should we pass to have a correct background for both hemispheres, and if I am missing something could we add an example?

Reply:
> so if we pass a numpy array or surface data file we would use the same bg map for both hemispheres which doesn't make sense

Ohhh, now I see what you mean! Sorry, I had forgotten that `view_img_on_surf()` doesn't take `hemi` as an argument, and therefore the background map should work for both hemispheres.
I suggest we either:
1. remove the possibility to tweak background maps here (and simply use the curvature map)
2. rename `bg_map` to `bg_maps`. It would default to `"auto"`, but could also take a list of two background maps (either paths or np arrays)

I'm in favour of option 2, although I don't really like the fact that this introduces a new argument.

Reply:
Option 2 seems better to me too.